Skype/UCWA/simultaneousRingToTeam_ref.md

```diff
@@ -1,3 +1,6 @@
+---
+title: simultaneousRingToTeam resource description
```

Comment:
Change "resource description" to "reference"

Reply:
done.